PreFreeSurfer/scripts/BrainExtraction_FNIRTbased.sh

```diff
@@ -99,12 +130,62 @@ echo " " >> $WD/log.txt
 
 ########################################## DO WORK ##########################################
 
-
 # Register to 2mm reference image (linear then non-linear)
-verbose_echo " ... linear registration to 2mm reference"
-${FSLDIR}/bin/flirt -interp spline -dof 12 -in "$Input" -ref "$Reference" -omat "$WD"/roughlin.mat -out "$WD"/"$BaseName"_to_MNI_roughlin.nii.gz -nosearch
-verbose_echo " ... non-linear registration to 2mm reference"
-${FSLDIR}/bin/fnirt --in="$Input" --ref="$Reference2mm" --aff="$WD"/roughlin.mat --refmask="$Reference2mmMask" --fout="$WD"/str2standard.nii.gz --jout="$WD"/NonlinearRegJacobians.nii.gz --refout="$WD"/IntensityModulatedT1.nii.gz --iout="$WD"/"$BaseName"_to_MNI_nonlin.nii.gz --logout="$WD"/NonlinearReg.txt --intout="$WD"/NonlinearIntensities.nii.gz --cout="$WD"/NonlinearReg.nii.gz --config="$FNIRTConfig"
+if [ "$BrainExtract" = INVIVO ] ; then
```

Comment:
Since all these options are NHP specific, I wonder if it would be better code organization and usability to do a if not human conditional around this stuff since we don't really know if any of those additional options would actually work for humans (e.g., EXVIVO).